source/fundamentals/crud/read-operations/sort.txt

```diff
@@ -4,135 +4,382 @@ Sort Results
 
 .. default-domain:: mongodb
 
-Use ``sort()`` to change the order in which read operations return
-documents. The ``sort()`` instance method tells MongoDB to order returned documents by the
-values of one or more fields in a certain direction. To sort returned
-documents by a field in ascending (lowest first) order, use the static method ``Sorts.ascending()``, 
-specifying the field on which to sort. To sort in descending (greatest first) order instead, 
-use the static method ``Sorts.descending()``, specifying the field on which to sort. For example,
-you can sort on ``id`` or ``name`` fields. If you do not specify a sort, MongoDB does not 
-guarantee the order of query results.
-
-Follow the instructions in the examples below to insert data into
-a collection and perform a sort on the results of a query.
-Consider a collection containing documents that describe books. To
-insert this data into a collection, run the following operation:
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, we show you how to **sort** your results.
+
+Sorting orders the documents returned from your query by your specified 
+**sort criteria**. Sort criteria are the rules you give MongoDB that describe
```

Comment:
Watch out for dropping the preposition on indirect objects:
```suggestion
**sort criteria**. Sort criteria are the rules you give to MongoDB that describe
```

Reply:
Thanks for the note Chris. After looking around online, it seems like there is mixed opinion on whether or not non-pronoun indirect objects require a preposition. [This site](https://www.chompchomp.com/terms/indirectobject.htm) only recommends prepositions for pronoun indirect objects. This seems like the [difference between a prepositional phrase and an indirect object](really-learn-english.com/english-direct-object-indirect-object-and-object-of-the-preposition.html#:~:text=How%20can%20you%20tell%20if,followed%20by%20the%20direct%20object.)? Not sure, would appreciate discussion on this so that going forward I do not make this mistake.

Reply:
Not a big deal, but it reads a bit sloppy to me without when you have a direct object right there as well.

Reply:
Sounds good. My ear doesn't pick up on the sloppiness, so I'll try and keep an eye out for direct object oddities going forward.